### PR TITLE
FeatureVet clinic database: create animals table

### PR DESCRIPTION
**Pull request by hunter4466**
Used template to generate your repo.

- [x] Created a database named vet_clinic.
- [x] Created a table named animals with the following columns:
- [x] id: integer
- [x] name: string
- [x] date_of_birth: date
- [x] escape_attempts: integer
- [x] neutered: boolean
- [x] weight_kg: decimal

Inserted the following data:

- [x] Animal: His name is Agumon. He was born on Feb 3rd, 2020, and currently weighs 10.23kg. He was neutered and he has never tried to escape.
- [x] Animal: Her name is Gabumon. She was born on Nov 15th, 2018, and currently weighs 8kg. She is neutered and she has tried to escape 2 times.
- [x] Animal: His name is Pikachu. He was born on Jan 7th, 2021, and currently weighs 15.04kg. He was not neutered and he has tried to escape once.
- [x] Animal: Her name is Devimon. She was born on May 12th, 2017, and currently weighs 11kg. She is neutered and she has tried to escape 5 times.

Wrote queries for the following:

- [x] Found all animals whose name ends in "mon".
- [x] Listed the name of all animals born between 2016 and 2019.
- [x] Listed the name of all animals that are neutered and have less than 3 escape attempts.
- [x] Listed date of birth of all animals named either "Agumon" or "Pikachu".
- [x] Listed name and escape attempts of animals that weigh more than 10.5kg
- [x] Found all animals that are neutered.
- [x] Found all animals not named Gabumon.
- [x] Found all animals with a weight between 10.4kg and 17.3

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -1,5 +1,6 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO `vet_clinic`.`animals` (`name`, `date_of_birth`, `escape_attempts`, `neutered`, `weight_kg`) VALUES ('Agumon', '2020-02-03', '0', '1', '10.23');
+INSERT INTO `vet_clinic`.`animals` (`name`, `date_of_birth`, `escape_attempts`, `neutered`, `weight_kg`) VALUES ('Gabumon', '2018-11-15', '2', '1', '8.00');
+INSERT INTO `vet_clinic`.`animals` (`name`, `date_of_birth`, `escape_attempts`, `neutered`, `weight_kg`) VALUES ('Pikachu', '2021-01-07', '1', '0', '15.04');
+INSERT INTO `vet_clinic`.`animals` (`name`, `date_of_birth`, `escape_attempts`, `neutered`, `weight_kg`) VALUES ('Devimon', '2017-05-12', '5', '1', '11.00');
```
Comment (Davidosky007) [inline data.sql]: * Kindly add the correct values for neutered which should be either `true` or `false` [Refer to the requirement ](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_create_tables.md#:~:text=escape_attempts%3A%20integer-,neutered%3A%20boolean,-weight_kg%3A%20decimal)

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM vet_clinic.animals WHERE name like '%mon'
+SELECT name FROM vet_clinic.animals WHERE date_of_birth >= '20160101' AND date_of_birth <= '20191231';
+SELECT name FROM vet_clinic.animals WHERE neutered = "1" AND escape_attempts < 3;
+SELECT date_of_birth FROM vet_clinic.animals WHERE name = 'Agumon' OR name = 'Pikachu';
+SELECT name, escape_attempts FROM vet_clinic.animals WHERE weight_kg > "10.50";
+SELECT * FROM vet_clinic.animals WHERE neutered = "1";
```
Comment (Davidosky007) [inline queries.sql]: * I suggest you use the proper data type for neutered as it is a `BOOLEAN` and not an `integer` [Check requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_create_tables.md#:~:text=escape_attempts%3A%20integer-,neutered%3A%20boolean,-weight_kg%3A%20decimal)

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
## Hi @hunter4466 👋, 

Good job so far! 💪 
* Nice PR ✔️ 
* Descriptive commit messages ✔️ 
* Well written Readme file ✔️ 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by oxenprogrammer — APPROVED** [review]
Hi @hunter4466 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**